src/Engines/Nunjucks.js

```diff
@@ -225,13 +227,19 @@ class Nunjucks extends TemplateEngine {
   }
 
   async compile(str, inputPath) {
+    let needsCompile = (str.indexOf("{{") != -1) || (str.indexOf("{%") != -1);
```

Comment:
Oh nice! I do wonder if Nunjucks specifically allows you to change those delimiters, some template languages do

Reply:
Will check.

Reply:
Indeed, it seems delimiters can be customized. Will try to defend against this:

http://mozilla.github.io/nunjucks/api.html#customizing-syntax

Reply:
Reading [`src/Engines/Nunjucks.js`](https://github.com/11ty/eleventy/blob/master/src/Engines/Nunjucks.js), it appears that `njkEnv` is exported from the public API, which would allow calls to `.configure()` outside the explicitly surfaced API and there's no way to set these through the various `UserConfig` attributes AFAICT (`nunjucksFilters`, `nunjucksAsyncFilters`, `nunjucksTags`, etc.)

What's the project policy on this sort of configuration? How defensive are you to monkey-patched, out-of-band configuration?

I *think* it's possible to reach into the NJK engine to figure out the delimiters, and will move to that if I can make it work, but knowing how "out of bounds" it is to rewire template engines outside the explicit API would be helpful in future.

Reply:
Added defensive code here:

https://github.com/11ty/eleventy/pull/1273/commits/b709096715eafe9cdffa8ce6cb86298450773e65#diff-466c0bd9f535df3ab417f301360293b9R230